lib/image_optim/runner/option_parser.rb

```diff
@@ -204,6 +204,19 @@ def wrap_regex(width)
   end
 
   op.separator nil
+  op.separator '  Caching:'
```

Comment:
Please move this section higher, after main options and before disabling workers